### PR TITLE
Migrate away from deprecated sass import system

### PR DESCRIPTION
**Pull request by Stormheg**
As discussed with you @thibaudcolas

### CODE REVIEW

**Comment by Stormheg** [comment]
Looks like I tagged some people in commit messages. Sorry! Didn't know this was a thing GitHub did.

File: wagtailio/static/sass/main.scss [hunk 1]
```diff
@@ -1,96 +1,96 @@
...
-@import 'vendor/normalize';
-@import 'vendor/code-highlighting';
+@use 'vendor/normalize';
+@use 'vendor/code-highlighting';
 
 // Abstracts
-@import 'abstracts/variables';
-@import 'abstracts/mixins';
+@use 'abstracts/variables';
+@use 'abstracts/mixins';
```
Comment (thibaudcolas) [inline wagtailio/static/sass/main.scss:7]:
```suggestion
@use 'abstracts/variables' as *;
@use 'abstracts/mixins' as *;
```

**Review by thibaudcolas — COMMENTED** [review]
With this change, it should be possible to skip all other `.scss` changes.

**Comment by thibaudcolas** [comment]
Thank you @Stormheg!

**Pull request merged**